Analytics.Xamarin.Pcl/Client.cs

```diff
@@ -232,9 +232,9 @@ public Task Group(string userId, string groupId, IDictionary<string, object> tra
 		/// that it is in human readable form. For example, "Bought T-Shirt"
 		/// or "Started an exercise"</param>
 		///
-		public Task Track(string userId, string eventName)
+		public async Task<bool> Track(string userId, string eventName)
```

Comment:
I don't think you needed to change these. Just doing `Task`->`Task<bool>` is enough, which is what you did for most of the others it looks like. I'd prefer that it matched the other so the file was consistent throughout but it isn't a huge deal.